Packs/PICUS/Integrations/PICUS/PICUS.py

```diff
@@ -0,0 +1,865 @@
+import json
+import os
+import traceback
+
+import demistomock as demisto  # noqa: F401
+import requests
+from CommonServerPython import *  # noqa: F401
+# from requests.exceptions import ConnectionError, HTTPError
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBAL VARIABLES'''
+VERIFY_SSL = not demisto.params().get('insecure', False)
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+if not demisto.params().get('proxy', False):
+    del os.environ['HTTP_PROXY']
+    del os.environ['HTTPS_PROXY']
+    del os.environ['http_proxy']
+    del os.environ['https_proxy']
+
+
+def test_module() -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    try:
+        SERVER = demisto.params().get('server')[:-1] if str(demisto.params().get('server')).endswith('/') \
+            else demisto.params().get('server')
+        CREDENTIALS = demisto.params().get('apikey')
+        # AUTH_HEADERS = {'Content-Type': 'application/json'}
+
+        params = "Bearer " + CREDENTIALS
+        headers = {"X-Refresh-Token": params}
+        endpoint = "/authenticator/v1/access-tokens/generate"
+        url = SERVER + endpoint
+        req = requests.post(url, headers=headers, verify=VERIFY_SSL)
+        parsed = json.loads(req.content)
+        data = parsed['data']
+        accessToken = data['access_token']
+
+    except DemistoException as e:
+        if ('Forbidden' in str(e)) or not(accessToken):
```

Comment:
1. You shouldn't catch `DemistoException` alone, but general `Exception`, since you have request and json parsing here.
2. In any case that you enter this exception block `accessToken` will not be defined, then the `not(accessToken)` part will always be True.

Reply:
Thanks for your supports @teizenman, It seems everthing is done.